README.md

```diff
@@ -60,19 +60,21 @@ pip install src/spacy
 Some components or modules in forte may require some [extra requirements](https://github.com/asyml/forte/blob/master/setup.py#L45):
 
 
-* `pip install forte[data_aug]`: Install packages required for [data augmentation modules](https://github.com/asyml/forte/tree/master/forte/processors/data_augment).
```

Comment:
any reasons that we re-oder these lines? The previous order is based on whether they are actively developed by us.

Reply:
I wanted to maintain the same order as those modules in `setup.py` so that I could check them in parallel. I can rearrange the order in `setup.py` instead.